### ISSUE TITLE
Issues with files named "display"

### ISSUE THREAD

**Issue by PRudykIteo** [nopSolutions/nopCommerce-Docs#141]
Recently I found out that two images are broken in documentation: [Manufacturers](https://docs.nopcommerce.com/en/running-your-store/catalog/manufacturers.html#display) and [Categories](https://docs.nopcommerce.com/en/running-your-store/catalog/categories.html#display).
Frankly, image was not displayed, but it didn't show any errors in console. To test it out, I have pasted image URL to address bar in my browser and it redirected me to `/en/index.html`.
It seems, that files which have text `display` in their names are messing with the last redirection rule in `web.config`:
```
<rule name="allotherpages" stopProcessing="true">
       <match url="^(pages/viewpage\.action.*)|(display.*)$" />
       <action type="Redirect" appendQueryString="false" url="/en/index.html" redirectType="Permanent" />
</rule>
```

I think that this is caused by small error in regex.
I propose to change it to:
`(pages\/viewpage\.action.*)|^(display.*)$` [Test](https://regex101.com/r/ctggfL/2)
or equivalently
`^(?>(pages\/viewpage\.action.*)|(display.*))$` [Test](https://regex101.com/r/U0sOcx/2)
The main difference is the `^` character.

I'd make a PR with that but I'm not sure what was the original problem, so You may want to test it out on some old links before making any changes.

**Comment by AndreiMaz** [comment]
@skoshelev Just ensure that old URLs from confluence work fine

**Comment by skoshelev** [comment]
It looks like all working fine now

Closed #141

**Comment by PRudykIteo** [comment]
Seems that @exileDev fixed it in a092d057633232bda4b55a487e2810835c1bd3ba
